examples/sqla.py

```diff
@@ -29,17 +30,28 @@ def main():
     settings = {}
     settings.update(options.group_dict(None))
     settings.update(tornado_jsonapi.handlers.not_found_handling_settings())
+    settings.update({'jsonapi_limit': 12})
 
     engine = create_engine('sqlite:///:memory:', echo=settings['debug'])
     Base.metadata.create_all(engine)
     Session = sessionmaker(bind=engine)
+    
+    s = Session()
+    for i in range(1,16):
+        p = Post()
+        p.author = "Author %d" % i
+        p.text = "Text for %d" % i
+        s.add(p)
+    s.commit()
+
+    postResource = tornado_jsonapi.resource.SQLAlchemyResource(Post, Session)
+    postResource.blacklist.append(Post.hideMe)
```

Comment:
Maybe it would be better to use simple strings as field designators here.

Reply:
Strangely enough both work. I have added a strings example.